crates/bevy_input/src/input.rs

```diff
@@ -71,3 +71,79 @@ where
         self.just_released.iter()
     }
 }
+
+mod test {
```

Comment:
can we annotate these modules with `#[cfg(test)]`? its not strictly required (because the functions inside the module are labeled as tests, which means they won't be included in the module for normal builds), but its a good way to ensure the module isn't compiled normally

Reply:
Yes that's a good idea!